File: PHP/UberBlack.php
<?php
require_once('Car.php');
class UberBlack extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __constructor($license, $driver, $typeCarAccepted, $seatsMaterial){
        parent::__constructor($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}
?>

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    
    public UberBlack(String license, Account driver, Integer passengers, String brand, String model, Map<String, Map<String, Integer>> typeCarAccepted, ArrayList<String> seatsMaterial){
        super(license, driver, passengers, brand, model);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
    void printDataUberBlack(){
        if(passengers == 4){
            System.out.println("Type Uber: Uber X. License: " + license + " Driver: " + driver.name + " Brand: " + brand + " Model: " + model + ". Passengers: " + passengers);
        }else{
            System.out.println("Introce el número correcto de pasajeros");
        }
        
    }
}
